script/src/run-dev/index.ts

```diff
@@ -169,7 +169,7 @@ export async function main(
             npmBinCommand({
               name: `${name}:run`,
               command:
-                'while [ ! -f build/index.js ]; do echo "Waiting for build…"; sleep 1; done; nodemon --watch build --delay 1 --exitcrash --exec NODE_ENV=development node build/index.js',
+                `while [ ! -f build/index.js ]; do echo "Waiting for build…"; sleep 1; done; nodemon --watch build --delay 1 --exitcrash --exec NODE_ENV=development VX_MACHINE_TYPE=${frontend} node build/index.js`,
```

Comment:
`VX_MACHINE_TYPE` is specified in prod through [this line](https://github.com/votingworks/vxsuite-complete-system/blob/a6f272bc74684807c87d80be06926550fe61ab4a/config/read-vx-machine-config.sh#L38)